File: src/main/java/seremis/geninfusion/soul/gene/newAI/creeperSwell/GeneAICreeperSwell.java
package seremis.geninfusion.soul.gene.newAI.creeperSwell;

import seremis.geninfusion.api.soul.IAllele;
import seremis.geninfusion.api.soul.lib.Genes;
import seremis.geninfusion.soul.Gene;
import seremis.geninfusion.soul.MasterGene;
import seremis.geninfusion.soul.allele.AlleleBoolean;

public class GeneAICreeperSwell extends MasterGene {

    public GeneAICreeperSwell() {
        addControlledGene(Genes.GENE_AI_CREEPER_SWELL_INDEX);
    }

    @Override
    public Class<? extends IAllele> possibleAlleles() {
        return AlleleBoolean.class;
    }
}

File: src/main/java/seremis/geninfusion/soul/gene/newAI/GeneAICreeperSwell.scala
package seremis.geninfusion.soul.gene.newAI

import seremis.geninfusion.api.soul.lib.Genes
import seremis.geninfusion.soul.MasterGene

//If this is true, the entity can explode!
class GeneAICreeperSwell extends MasterGene(classOf[Boolean]) {
    addControlledGene(Genes.GeneAICreeperSwellIndex)
}
